lib/broccoli/fastboot-config.js

```diff
@@ -104,7 +104,9 @@ FastBootConfig.prototype.readAssetManifest = function() {
     var assetMap = JSON.parse(fs.readFileSync(assetMapPath));
     return assetMap;
   } catch (e) {
-    // No asset map was found, proceed as usual
+    if (assetMapPath) {
```

Comment:
Per the discussion at this week's meeting:

This warning should only be emitted if `assetRev && assetRev.options && assetRev.options.enabled`.